src/applications/facility-locator/components/markers/FacilityMarker.jsx

```diff
@@ -0,0 +1,25 @@
+import DivMarker from './DivMarker';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+class FacilityMarker extends Component {
```

Comment:
Since we are unable to change any of the interaction and markup in the Leaflet-Mapbox map, I think it may be wise to document the accessibility issues within that area separately using a Github label, 508-issues-leaflet-mapbox. This way we have documentation for the VA 508 Office, that we're aware, but are unable to address without the vendor addressing.  I checked with @mmiddaugh and @sshein and got 👍 .